docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -22,6 +22,7 @@ it is supported for production use.
 * [Running the Repository Creation Utility to set up your database schema](#running-the-repository-creation-utility-to-set-up-your-database-schema)
 * [Create a Kubernetes secret with the RCU credentials](#create-a-kubernetes-secret-with-the-rcu-credentials)
 * [Creating a SOA domain](#creating-a-soa-domain)
+* [Monitoring a SOA domain](#monitoring-a-soa-domain)
 
 
 #### Introduction
```

Comment:
Please remove "Starting with the 2.2.1 release, " from line 30.

Reply:
Done.